src/styles/stars.css

```
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  text-decoration: none;
+  background-image: url(/images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(n),
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5),
+.stars--5 .stars__star:nth-child(n+6) {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
```

all you need is to change `background-image` here, don't repeat yourself
```suggestion
  background-image: url(/images/star.svg);
```

---

oh... yeah, sorry didn't notice it... Thank you for your hints and help!